packages/base/src/panelview/rightpanel.tsx

```diff
@@ -23,6 +24,17 @@ export class RightPanelWidget extends SidePanel {
     });
 
     this.addWidget(properties);
+
+    const filterPanel = new FilterPanel({
```

Comment:
In a subsequent PR, we should probably not show this panel for some layers like Rasters. Also, it shows up when selecting a source from the source panel and in that case it does not make sense to show the filters.

Reply:
I added a little check so the filter options are only displayed for a `VectorLayer`